src/class-fields/computed-name-empty-var-empty-function.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Computed name var not initialized, empty function
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  ['a']; b(){}
+
+//- assertions
+  assert.sameValue(this.a, undefined);
+  verifyEnumerable(this, "a");
+  verifyWritable(this, "a");
+  verifyConfigurable(this, "a");
+
+  assert.sameValue(typeof Object.getPrototypeOf(this).b, "function");
+  verifyNotEnumerable(Object.getPrototypeOf(this), "b");
+  verifyConfigurable(Object.getPrototypeOf(this), "b");
```

Comment:
> OK, it sounds like the test is valid, you're just making cleanup suggestions. Is that accurate? 

100% accurate. I'm just looking for simplification. 

> For context, these tests are really aiming to get at whether the syntax is handled correctly. Verifying the properties is just meant as a smoke test that something vaguely correct happened. Tests for all the semantic details are expected to come separately. I'm not sure if that affects this review at all.

I understood that and it's exactly why I wanted to simplify the assertions. We're completely on the same page.